.env.development

```diff
@@ -110,19 +104,12 @@ DEFAULT_COUNTRY=IN
 HELP_SCREEN_YOUTUBE_PASSPORT_URL="https://youtu.be/aktZ1yTdDOA"
 HELP_SCREEN_YOUTUBE_TRAINING_URL="https://youtu.be/MC_45DoRw2g"
 HELP_SCREEN_YOUTUBE_VIDEO_URL="https://youtu.be/nHsQ06tiLzw"
-HELP_SCREEN_SUPPORT_GROUP_URL="http://localhost:3000"
-HELP_SCREEN_SUPPORT_GROUP_NUMBER="+00 0000000000"
-HELP_SCREEN_SUPPORT_GROUP_ICON="icon_whatsapp_icon.svg"
 
 IHCI_ORGANIZATION_UUID=89008920-021e-4116-b74c-8edf86180dd6
 OWNER_EMAILS='Owner <owner@test.com>, Another Owner <another-owner@test.com>'
 PROGRAM_INCEPTION_DATE=2018-01-01
 SUPERVISOR_EMAILS='Supervisor <supervisor@test.com>'
 
-SEED_GENERATED_ADMIN_PASSWORD='Resolve2SaveLives'
-SEED_GENERATED_ACTIVE_USER_ROLE="Seed User | Active"
-SEED_GENERATED_INACTIVE_USER_ROLE="Seed User | Inactive"
-
```

Comment:
`SEED_GENERATED_ADMIN_PASSWORD` is unused. Moved the others to the `.env.seed` files.